lib/samples/show_realistic_light_and_shadows/README.md

```diff
@@ -0,0 +1,31 @@
+# Show realistic light and shadows
+
+Show realistic lighting and shadows for a given time of day.
+
+![Image of show realistic light and shadows](show_realistic_light_and_shadows.png)
+
+## Use case
+
+You can use realistic lighting to evaluate the shadow impact of buildings and utility infrastructure on the surrounding community. This could be useful for civil engineers and urban planners, or for events management assessing the impact of building shadows during an outdoor event.
+
+## How to use the sample
+
+Select one of the three lighting options to show that lighting effect on the SceneView. Select a time of day from the slider (based on a 24hr clock) to show the lighting for that time of day in the SceneView.
```

Comment:
```suggestion
Select one of the three lighting options to show that lighting effect on the scene view. Select a time of day from the slider (based on a 24hr clock) to show the lighting for that time of day in the scene view.
```
Since our class is `ArcGISSceneView` I think it's cleaner to reference as the "concept" rather than the class.